routes/v0/contacts.js

```diff
@@ -8,15 +8,15 @@ const CaseController = require('../../controllers/case-controller')(Case, sequel
 const contactsRouter = Router();
 
 contactsRouter.get('/', async (req, res) => {
-  console.log('>>> req.accountSid: ', req.accountSid); // TODO: remove log, is just for showing how to use the accountSid
```

Comment:
This doesn't hurt, so I would suggest leaving it for debugging but is your call ;)